platforms/ios/framework/Tangram-es.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.documentation_url = 'https://mapzen.com/documentation/tangram/ios-framework/0.9.0/'
 
   s.source = {
-    :http => "http://ios.mapzen.com/tangram-releases/tangram-release-#{s.version}.zip"
+    :http => "https://dl.bintray.com/tangrams/cocoapods/:#{s.version}-release.zip"
```

Comment:
We should update the homepage above, to maybe point to github repo url. Similarly the documentation link to github.com/tangrams/tangram-docs.

Actually a few things here:
- author: tangram-es team?, with a shared email address?
- social_media_url:? tangramJS twitter handle?